common-service-core/src/main/java/com/ca/mfaas/message/core/AbstractMessageService.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.message.core;
+
+import com.ca.mfaas.message.storage.MessageTemplateStorage;
+import com.ca.mfaas.message.template.MessageTemplate;
+import com.ca.mfaas.message.template.MessageTemplates;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Abstract class which implements the {@link MessageService} interface.
+ * It creates messages from the file, validate and add them to the {@link MessageTemplateStorage}.
```

Comment:
"... validate**s** and add**s**...

Reply:
Yes you're right, it's "validates and adds"